src/App.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import "./App.css";
+
+const Form = props => {
+    console.log(props);
+    const [team, setTeam] = useState({ name: "", email: "", role:""});
```

Comment:
Is this form submitting an entire team or just a member addition to the team state stored in **App.js**? Naming the state like this might make things a bit confusing.